domain/personallearningplan/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/personallearningplan/service/GoalService.kt

```diff
@@ -64,7 +64,7 @@ class GoalService(
 
   fun getGoals(getGoalsDto: GetGoalsDto): GetGoalsResult {
     return goalPersistenceAdapter.getGoals(getGoalsDto.prisonNumber)
-      ?.filter { getGoalsDto.status == null || getGoalsDto.status == it.status }
+      ?.filter { getGoalsDto.statuses.isNullOrEmpty() || getGoalsDto.statuses.contains(it.status) }
```

Comment:
You can also do `it.status in getGoalsDto.statuses`

Reply:
Oh yeah, `in` is a nicer syntax 👍 
Am changing it to that 👍